packages/experiments/src/components/Chiclet/ChicletCard.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  KeyCodes,
+  css,
+  classNamesFunction
+} from '../../Utilities';
+import {
+  IChicletCardProps,
+  IChicletStyles,
+  IChicletStyleProps
+} from './Chiclet.types';
+import { mergeStyles } from '../../Styling';
+import {
+  getClassNames
+} from './Chiclet.styles';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
+import { IconButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+import { Image } from 'office-ui-fabric-react/lib/Image';
+import { TestImages } from 'office-ui-fabric-react/src/common/TestImages';
+import { FocusZone } from 'office-ui-fabric-react/lib/FocusZone';
+import { ChicletTestImages } from '../../common/TestImages';
+
+export class ChicletCard extends BaseComponent<IChicletCardProps, any> {
+  private _classNames: IChicletStyles = {};
+
+  public render() {
+    const { styles: customStyles, title, ogType, description, image, imageType, imageWidth, imageHeight, url, onClick, onClickHref, className, actions, theme } = this.props;
+    const actionable = (onClick || onClickHref) ? true : false;
+
+    this._classNames = getClassNames(theme!, customStyles);
+
+    // if this element is actionable it should have an aria role
+    const role = actionable ? (onClick ? 'button' : 'link') : undefined;
+    const tabIndex = actionable ? 0 : undefined;
+
+    var preview = this._renderPreviewImage(image, imageHeight, imageWidth, ogType);
+
+    return (
+      <div
+        tabIndex={ tabIndex }
+        role={ role }
+        onKeyDown={ actionable ? this._onKeyDown : undefined }
+        onClick={ actionable ? this._onClick : undefined }
+        className={
+          css('ms-ChicletCard', className) }
+      >
+        <div
+          className={ mergeStyles(this._classNames.preview) }
+        >
+          { image ?
+            preview :
+            (<Image
+              src={ TestImages.documentPreviewTwo }
+              role='presentation'
+              alt=''
+            />)
+          }
+        </div>
+        <div
+          className={ mergeStyles(this._classNames.info) }
+        >
+          <div
+            className={ mergeStyles(this._classNames.title) }
+          >
+            { title ? title : "Placeholder" }
+          </div>
+          <div
+            className={ mergeStyles(this._classNames.link) }
+          >
+            { url ? url : "https://onedrive.com/files/v-lygi/39192908430" }
+          </div>
+          { actions ? this._renderFooter(actions) : (null) }
+        </div>
+      </div>
+    );
+  }
+
+  private _renderPreviewImage(imageUrl?: string, imageHeight?: string, imageWidth?: string, ogType?: string): React.ReactElement<React.HTMLAttributes<HTMLDivElement>> {
+    const image = (
+      <Image
+        width={ imageWidth }
+        height={ imageHeight }
+        src={ TestImages.documentPreview }
+        role='presentation'
+        alt=''
```

Comment:
We should try to parse and provide `alt` text at some point for users with accessibility needs. Otherwise a person with vision problems using assistive technology won't know what this image is of. It looks like OG offers `og:image:alt` for this reason.

https://webaim.org/techniques/alttext/#intro

FYI - I use to help find common a11y issues if you find it helpful, http://wave.webaim.org/extension/